okhttp/src/main/java/okhttp3/RequestBody.kt

```diff
@@ -91,11 +91,12 @@ abstract class RequestBody {
   companion object {
 
     /**
-     * Returns a new request body that transmits [content]. If [contentType] is non-null
-     * and lacks a charset, this will use UTF-8.
+     * Returns a new request body that transmits this string. If [contentType] is non-null and lacks
+     * a charset, this will use UTF-8.
      */
     @JvmStatic
-    fun create(contentType: MediaType?, content: String): RequestBody {
+    @JvmName("create")
+    fun String.toRequestBody(contentType: MediaType? = null): RequestBody {
       var charset: Charset = UTF_8
       var finalContentType: MediaType? = contentType
       if (contentType != null) {
```

Comment:
4.1: this logic of resolving a final MediaType with charset seems like it should be part of MediaType.  Especially if it's something we deem important enough to do automatically for users of the RequestBody API.

Reply:
`mediaTpe.copy(charset=UTF_8)` ? Dig.

(For follow up!)